applications/nrf_desktop/src/modules/dvfs.c

```diff
@@ -56,6 +60,14 @@ static struct dvfs_retry_work_data {
 	uint8_t dfvs_retries_cnt;
 } dvfs_retry_data;
 
+struct dvfs_state_timeout_data {
+	struct k_work_delayable dvfs_timeout_work;
```

Comment:
Since the struct is already called `dvfs_state_timeout_data`, consider using shorter names for members (e.g. `work` and `timeout_ms`. We could also name the struct; `dvfs_state_timeout` (`_data` suffix may be omitted as it introduces no information)

Reply:
Similar could be considered for `dvfs_retry_work_data`